packages/app/src/components/page-information-block/useful-links.tsx

```diff
@@ -0,0 +1,148 @@
+import { useIntl } from '~/intl';
+import { asResponsiveArray } from '~/style/utils';
+import { isAbsoluteUrl } from '~/utils/is-absolute-url';
+import { Link } from '~/utils/link';
+
+interface usefulLinksProps {
+  links: {
+    title: string;
+    href: string;
+  }[];
+}
+
+export function UsefulLinks({ links }: usefulLinksProps) {
+  const { siteText } = useIntl();
+
+  return (
+    <Box
+      borderTop="1px solid"
+      borderTopColor="border"
+      width="100%"
+      pt={3}
+      mt={3}
+    >
+      <InlineText
+        mb={2}
+        fontSize={2}
+        fontWeight="bold"
+        css={css({ display: 'block' })}
+      >
+        {siteText.informatie_header.handige_links}
+      </InlineText>
+      <OrderedList>
+        {links.map((link, index) => (
+          <ListItem key={index}>
+            {isAbsoluteUrl(link.href) ? (
+              <ExternalLink href={link.href}>
+                <TitleWithIcon title={link.title} icon={<ExternalLinkIcon />} />
+              </ExternalLink>
+            ) : (
+              <>
+                <Link href={link.href} passHref>
+                  <a>
+                    <TitleWithIcon
+                      title={link.title}
+                      icon={<ChevronLargeIcon />}
+                    />
+                  </a>
+                </Link>
+              </>
+            )}
+          </ListItem>
+        ))}
+      </OrderedList>
+    </Box>
+  );
+}
+
+function TitleWithIcon({ title, icon }: { title: string; icon?: JSX.Element }) {
+  const splittedWords = title.split(' ');
+
+  return (
+    <>
+      {splittedWords.map((word, index) => (
+        <InlineText
+          key={index}
+          css={css({
+            whiteSpace: 'pre-wrap',
+            fontFamily: 'body',
+            fontSize: '1rem',
+          })}
+        >
+          {splittedWords.length - 1 === index ? (
+            <InlineText
+              css={css({
+                display: 'flex',
+                position: 'relative',
+                flexWrap: 'wrap',
+              })}
+            >
+              {word}
+              <IconContainer>{icon}</IconContainer>
+            </InlineText>
+          ) : (
+            `${word} `
+          )}
+        </InlineText>
+      ))}
+    </>
+  );
+}
+
+const OrderedList = styled.ol(
+  css({
+    display: 'flex',
+    flexWrap: 'wrap',
+    p: 0,
+    m: 0,
+    listStyle: 'none',
+  })
+);
+
+const ListItem = styled.li(
+  css({
+    mb: 2,
+    pr: 4,
+    flexGrow: 1,
+    width: asResponsiveArray({ _: '100%', md: '50%' }),
+
+    // Remove the margin of the last 2 elements
+    '&:nth-last-of-type(-n+2)': {
+      mb: asResponsiveArray({ md: 0 }),
+    },
+
+    '&:last-of-type': {
+      mb: 0,
+    },
+
+    a: {
+      display: 'inline-flex',
+      flexWrap: 'wrap',
+      textDecoration: 'none',
+
+      '&:hover': {
+        textDecoration: 'underline',
+      },
+    },
+  })
+);
+
+const IconContainer = styled.div(
```

Comment:
Better not to render divs inside spans
```suggestion
const IconContainer = styled.span(
```
this might need an additional display style though (e.h. display:inline-block) you'll figure out whether its necessary or not.